crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -385,6 +385,13 @@ where
             // If the checkpoint of any stage is less than the checkpoint of the first stage,
             // retrieve and return the block hash of the latest header and use it as the target.
             if stage_checkpoint < first_stage_checkpoint {
+                warn!(
```

Comment:
This check is after a full pipeline run I assume?

Reply:
no, this happens only on start up